apps/jobs/admin.py

```diff
@@ -2,10 +2,8 @@
 
 from base.admin import TimeStampedAdmin
 
-from .models import Submission
```

Comment:
Hey, we are just removing submission app and not `Submission` model, So this file wont be touched at all.